servers-management/dell/ansible/inventory.yaml

```diff
@@ -0,0 +1,19 @@
+idrac:
+  hosts:
+      openshift-qe-047-drac.mgmt.arm.eng.rdu2.redhat.com:
+      openshift-qe-048-drac.mgmt.arm.eng.rdu2.redhat.com:
+      openshift-qe-049-drac.mgmt.arm.eng.rdu2.redhat.com:
+      openshift-qe-050-drac.mgmt.arm.eng.rdu2.redhat.com:
+      openshift-qe-051-drac.mgmt.arm.eng.rdu2.redhat.com:
+      openshift-qe-052-drac.mgmt.arm.eng.rdu2.redhat.com:
+      openshift-qe-053-drac.mgmt.arm.eng.rdu2.redhat.com:
+      openshift-qe-054-drac.mgmt.arm.eng.rdu2.redhat.com:
+      openshift-qe-055-drac.mgmt.arm.eng.rdu2.redhat.com:
+      openshift-qe-056-drac.mgmt.arm.eng.rdu2.redhat.com:
+      openshift-qe-057-drac.mgmt.arm.eng.rdu2.redhat.com:
+      openshift-qe-058-drac.mgmt.arm.eng.rdu2.redhat.com:
+  vars:
+    idrac_user: "user_here"
```

Comment:
should we remove this as managed by the idrac_vars.example.yaml file?

Reply:
removed